docker/base/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This is a base Docker image used by Wikia's MediaWiki app. It uses base PHP image provided by Docker with additional
 # production dependencies installed.
-FROM php:7.0.30-fpm-stretch
+FROM php:7.3.3-fpm-stretch
```

Comment:
7.3.7 on buster is out

Reply:
yeah, done in https://github.com/Wikia/app/pull/16873/commits/7f3999d4b05bfeccc44115b6d1bc1daaa9328de7